documentation/installation.md

```diff
@@ -40,6 +40,11 @@ pointing to it instead:
 
     ln -s <cloned_repo_path> deploy
 
+
+**Note:** Make sure to use Python 3.5 or 3.6 as the ansible-playbooks are not yet compatible 
```

Comment:
@ahmedbilal, in case you are wondering why we are still using Python 3.5 or 3.6 when 3.9 will be out soon - it is because Ubuntu 16.04 is the base OS supported by the Open edX platform (ships with Python 3.5.x) and we use the same base OS throughout our infrastructure to minimize compatibility issues. However, we had to upgrade Ocim to run on Python 3.6 (set up in production using `pyenv`, `pyenv-virtualenv`) as one of the dependencies we use required Python 3.6

Reply:
@lgp171188 Aah. Make sense now. Thanks :+1: